test_data/sample_input.csv

```diff
@@ -1,12 +1,12 @@
 table customer
-id; demand; hard_tw_begin; hard_tw_end; soft_tw_begin; soft_tw_end; service_time; suitable_vehicles
-1; 10; 0; 200; 50; 70; 10; 0; 1; 2
-2; 5; 10; 50; 10; 20; 5; 0; 3
-3; 1; 100; 110; 105; 110; 1; 1
-4; 2; 0; 20; 5; 15; 3; 2; 3
+id; demand; volume; weight; hard_tw_begin; hard_tw_end; soft_tw_begin; soft_tw_end; service_time; suitable_vehicles
```

Comment:
please, remove demand field. demand `<=>` transportation quantity

Reply:
yes, volume and weight determine the demand